docs/src/main/asciidoc/opentelemetry-logging.adoc

```diff
@@ -15,7 +15,7 @@ distributed logging for interactive web applications.
 
 [NOTE]
 ====
-- OpenTelemetry Logging is disabled by default.
+- OpenTelemetry Logging considered _tech preview_ and is disabled by default.
```

Comment:
```suggestion
- OpenTelemetry Logging is considered _tech preview_ and is disabled by default.
```

Reply:
Correct @sheilamjones. OpenTelemetry Logging is not yet part of the RHBQ.
Fixed.